new-frontiers-14/frontier-station-14
# Shuttle (New): FSB Nightlight

The Far Star Biotech Corporation is proud to present The Nightlight!

Preface: HUGE shout of thanks to Ego, Ego and Ego again in addition to Chef Mia, Fenrir, Grunk, Grug, Umami, Morgan and Mint Cherry for the feedback and support while this was being put together! Lots of chefs, hopefully, results in a good pot of soup.

## About the PR



This PR adds:

- The FSB Nightlight, a shuttle designed to provide a dining and bar experience with an emphasis on customer space and ease of use for the chef. 
- A new poster for the Nightlight, it's also a nightlight.

## Why / Balance


The inspiration for this shuttle came from discussing the state of kitchen shuttles with a number of fellow players. These discussions and ongoing feedback informed the following design goals before I set out.

1: The shuttle must have 3 docks to allow for additional guests.
2) As close to 50% as reasonable of the floor space must be reserved for guests and customization. This will make up the bulk of the space.
3) The kitchen must be created with an emphasis on comfortable and efficient movement and use. 
4) The ship should aim to be within reasonable scope and size of the Skipper. 
5) Lean into the new biorecomibinator whatsit as a tool to assist in maintaining shuttle endurance for the duration of the shift. It's a cool new tool and I want to see it get more use.
6) All of the usual FSB staples.

## How to test


- Double check the poster and description are good to go. Ensure it is part of the appropriate pools for a a contraband poster.

- Check shuttle air systems, power systems, button functionality and provide feedback on layout.

- Standard review of guidebook entry.

## Media


![ss+(2024-12-28+at+11 14 50)](https://github.com/user-attachments/assets/1d0fc2b4-0626-4e2c-bdb0-496fe12b2b40)
![ss+(2024-12-28+at+11 14 37)](https://github.com/user-attachments/assets/5e895e4e-2df9-45a9-af76-53d04a1408c3)

## Requirements



## Breaking changes


**Changelog**


🆑 
- add: FSB Nightlight service shuttle!
